QUEST_LV_0200.tsv

```diff
@@ -3253,40 +3253,40 @@ QUEST_LV_0200_20150323_003252	You may have to dedicate some flowers, or you may
+QUEST_LV_0200_20150323_003272	$Lower yourself in front of the Goddesses and leave everything to them.{nl}The Goddesses will bless the ones who truly confess their wrongdoings.{nl}
+QUEST_LV_0200_20150323_003273	$May the Goddesses remove your ordeals.{nl}-- Sanctum Manager : [Krivis Master] Hercus
+QUEST_LV_0200_20150323_003274	$Make contributions to appreciate the Goddesses.{nl}The blessing will be with you always.
+QUEST_LV_0200_20150323_003275	$Pray in front of the Goddesses.{nl}Sacrifice the vicious monsters to prove your belief.
+QUEST_LV_0200_20150323_003276	"I already knew that Pilgrim's Path was ruined with Naktis' curse.{nl}But I didn't know that even the Sanctums were corrupted.. It's such a shame.
+QUEST_LV_0200_20150323_003277	{memo X}It would be the best if I do it myself, but I can't leave this place behind.{nl}I want you to purify the Sanctums with this divine water.{nl}
+QUEST_LV_0200_20150323_003278	$I heard the congregation sent the disciples, but I guess it didn't turn out right.{nl}I think the everything will be alright since you will do it. But I am still uneasy.
+QUEST_LV_0200_20150323_003279	$The Vakarine Statue of the Forest of Prayer..{nl}Naktis.. How would have one ever known that he would ruin the lessons of Maven, the great bishop.
+QUEST_LV_0200_20150323_003280	$Anyways, we could at least do some emergency procedures.{nl}Collect Grey Gypsum Fragments and repair it with Dewdrops that are endowed with the blessings of the dawn stars.
+QUEST_LV_0200_20150323_003281	$I hope the Goddesses protect you.{nl}Not only the statues of the Goddesses, but the Sanctums also would have many problems.
+QUEST_LV_0200_20150323_003282	$By 'corrupted Sanctums', you mean the Forest of Prayer, right?{nl}I thought I had prepared a lot for that, but I guess Naktis' curse was even stronger.
+QUEST_LV_0200_20150323_003283	$I hid my Symbol of Faith in the Sanctums to prepare for times like this.{nl}That symbol enables you to find the monsters who have swallowed the Badges of the Sanctum.{nl}
+QUEST_LV_0200_20150323_003284	$Find the Badges of the Sanctum and burn them in front of the Sanctum.{nl}It may not return to how it used to be, but you will be able to purify it a bit.
+QUEST_LV_0200_20150323_003285	$The blessing that cleanses evil energy.{nl}Oh Goddesses, bless us with purified water.
+QUEST_LV_0200_20150323_003286	$A floral tribute to the Goddesses.{nl}The pleasure and joy of collecting flowers.
+QUEST_LV_0200_20150323_003287	$I was able to get one step closer to the Goddesses due to my contribution.{nl}May the blessing be with you always.
+QUEST_LV_0200_20150323_003288	$You know that Pilgrim's Way holds the lessons of Maven right?{nl}Naktis is so formidable. She was able to cast a curse on humans as if they originally sinned.
```

Comment:
True, but formidable is kind of like respect through strength/force/power.  This is more like an interesting irony that Naktis was able to pull off.